wiki/Ranking_Criteria/Difficulty_Naming/en.md

```diff
@@ -0,0 +1,137 @@
+
+### ![osu!](/wiki/shared/Osu.gif "osu!") osu!
+ 
+- ![Easy](/wiki/shared/diff/easy-s.png "Easy") Easy 
+- ![Normal](/wiki/shared/diff/normal-s.png "Normal") Normal
+- ![Hard](/wiki/shared/diff/hard-s.png "Hard") Hard
+- ![Insane](/wiki/shared/diff/insane-s.png "Insane") Insane
+- ![Extra](/wiki/shared/diff/expert-s.png "Extra") Extra
+
+### ![osu!taiko](/wiki/shared/Taiko.gif "osu!taiko") osu!taiko
+
+- ![Kantan](/wiki/shared/diff/easy-t.png "Kantan") Kantan
+- ![Futsuu](/wiki/shared/diff/normal-t.png "Futsuu") Futsuu
+- ![Muzukashii](/wiki/shared/diff/hard-t.png "Muzukashii") Muzukashii
+- ![Oni](/wiki/shared/diff/insane-t.png "Oni") Oni
+- ![Inner Oni / Ura Oni](/wiki/shared/diff/expert-t.png "Inner Oni / Ura Oni") Inner Oni / Ura Oni
+
+### ![osu!catch](/wiki/shared/Ctb.gif "osu!catch") osu!catch
+
+- ![Cup](/wiki/shared/diff/easy-c.png "Cup") Cup 
+- ![Salad](/wiki/shared/diff/normal-c.png "Salad") Salad
+- ![Platter](/wiki/shared/diff/hard-c.png "Platter") Platter
+- ![Rain](/wiki/shared/diff/insane-c.png "Rain") Rain
+- ![Overdose](/wiki/shared/diff/expert-c.png "Overdose") Overdose
+
+### ![osu!mania](/wiki/shared/Mania.gif "osu!mania") osu!mania
+
+- ![EZ](/wiki/shared/diff/easy-m.png "EZ") EZ 
+- ![NM](/wiki/shared/diff/normal-m.png "NM") NM
+- ![HD](/wiki/shared/diff/hard-m.png "HD") HD
+- ![MX](/wiki/shared/diff/insane-m.png "NX") MX
+- ![SC](/wiki/shared/diff/expert-m.png "SC") SC
+
+## Alternative
+
+The following variants are not commonly used, but they can also be used as a valid alternative way of spread difficulty naming.
+
+### ![osu!](/wiki/shared/Osu.gif "osu!") osu!
+
+- ![Beginner](/wiki/shared/diff/easy-s.png "Beginner") **Beginner**: in an ENHIX spread, it means the lowest level.
+- ![Advanced](/wiki/shared/diff/normal-s.png "Advanced") **Advanced**: in an ENHIX spread, it's a difficulty that is between Normal and Hard.
+- ![Hyper](/wiki/shared/diff/hard-s.png "Hyper") **Hyper**: in an ENHIX spread, it means a more difficult Hard (also known as Light Insane).
+- ![Extreme](/wiki/shared/diff/expert-s.png "Extreme") **Extreme**: uncommon, but it can be used to represent a difficulty tier above Extra/Expert.
+- ![Marathon](/wiki/shared/diff/expert-s.png "Marathon") **Marathon**: map that is over 5 minutes of Drain Time.
+
+### ![osu!taiko](/wiki/shared/Taiko.gif "osu!taiko") osu!taiko
+
+- ![Hell Oni](/wiki/shared/diff/expert-t.png "Hell Oni") **Hell Oni**: commonly accepted as a difficulty that is a tier above Inner Oni / Ura Oni.
+
+### ![osu!catch](/wiki/shared/Ctb.gif "osu!catch") osu!catch
+
+- ![Deluge](/wiki/shared/diff/expert-c.png "Deluge") **Deluge**: when there are two Overdose difficulties in a single set, the harder of the two will usually be called Deluge.
+
+## Songs from other Rhythm Games
+
+This subsection shows the difficulty naming for mapsets of a song that comes from another rhythm game. Using said rhythm game's method of difficulty naming is acceptable.
```

Comment:
Perhaps change to:
Using their respective style/method of difficulty naming is acceptable.